airbyte-integrations/connectors/source-freshdesk/unit_tests/test_source.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+import logging
+from source_freshdesk import SourceFreshdesk
+
+logger = logging.getLogger("test_source")
+
+
+def test_check_connection_ok(requests_mock, config):
+    json_resp = {
+        "primary_language": "en",
+        "supported_languages": [],
+        "portal_languages": []
+    }
+
+    requests_mock.register_uri("GET", "/api/v2/settings/helpdesk", json=json_resp)
+    ok, error_msg = SourceFreshdesk().check_connection(logger, config=config)
+
+    assert ok
+    assert not error_msg
+
+
+def test_check_connection_invalid_api_key(requests_mock, config):
+    responses = [{
+        "json": {
+            "code": "invalid_credentials",
+            "message": "You have to be logged in to perform this action."
+        },
+        "status_code": 401
+    }]
+
+    requests_mock.register_uri("GET", "/api/v2/settings/helpdesk", responses)
+    ok, error_msg = SourceFreshdesk().check_connection(logger, config=config)
+
+    assert not ok
```

Comment:
use single line statements/assertions `assert not ok and error_msg == ""`

Reply:
Updated in [4e2f437](https://github.com/airbytehq/airbyte/pull/12334/commits/4e2f4374641f906dfc9b9c69a77f02847186ef65).